addons/website_slides/controllers/main.py

```diff
@@ -908,11 +909,15 @@ def slide_preview(self, slide_id):
             slide.is_preview = not slide.is_preview
         return slide.is_preview
 
-    @http.route(['/slides/slide/send_share_email'], type='json', auth='user', website=True)
-    def slide_send_share_email(self, slide_id, email, fullscreen=False):
-        slide = request.env['slide.slide'].browse(int(slide_id))
-        result = slide._send_share_email(email, fullscreen)
-        return result
+    @http.route(['/slides/send_share_email'], type='json', auth='user', website=True)
+    def slide_send_share_email(self, emails, res_id, res_model, fullscreen=False):
+        for email in emails.split(','):
+            if not single_email_re.match(email.strip()):
+                return False
+        if res_model not in ['slide.channel', 'slide.slide']:
+            raise werkzeug.exceptions.NotFound()
+        slide = request.env[res_model].browse(int(res_id))
+        return slide._send_share_email(emails, fullscreen)
```

Comment:
Not sure I see the point of returning the mail_ids, you could call the method and simply return "True" as the caller does not seem to care.